automation/scripts/local.py

```diff
@@ -11,30 +11,43 @@ def main():
     parser = argparse.ArgumentParser(
         "A tool for running the automation locally"
     )
-    parser.add_argument("action", choices=["poll"])
+    parser.add_argument("action", choices=["poll", "migrate-meta"])
     parser.add_argument(
         "--table",
-        type=lambda val: getattr(tables, val.upper()),
         required=True,
         help="Which Airtable table to use",
     )
     parser.add_argument(
         "--live",
-        type=bool,
-        default=False,
+        action="store_true",
         help="Enables updating airtable records",
     )
 
     args = parser.parse_args()
 
     conf = config.load()
-    client = args.table.get_airtable_client(
-        conf.airtable, read_only=not args.live
-    )
+    table = tables.POLLABLE_TABLES[args.table](conf, read_only=not args.live)
 
     succeeded = True
     if args.action == "poll":
-        succeeded = client.poll_table(conf)
+        succeeded = table.poll_table()
+    elif args.action == "migrate-meta":
```

Comment:
This does the data migration copying the last seen status from the old `_meta` field used by the node automation to `_meta_last_seen_status` used by this repo.